src/main/scala/com/workday/warp/adapters/gatling/WarpFunSpec.scala

```diff
@@ -14,7 +15,7 @@ abstract class WarpFunSpec(val testId: String) extends GatlingHttpFunSpec with H
 
   def this() = this(DEFAULT_TEST_ID)
 
-  val controller: AbstractMeasurementCollectionController = new DefaultMeasurementCollectionController(this.canonicalName)
+  val controller: AbstractMeasurementCollectionController = WarpGuicer.getController(this.canonicalName, tags = List.empty)
```

Comment:
this is needed b/c if any external class extends this class **and** has modified/extended the warp-core schema, flyway migration issues will arise due to the warp-core schema still being instantiated.